app.cfg.example

```diff
@@ -111,10 +111,16 @@ deploy_permission = trz42
 #   and what submission parameters shall be used
 arch_target_map = { "linux/x86_64/generic" : "--constraint shape=c4.2xlarge", "linux/x86_64/amd/zen2": "--constraint shape=c5a.2xlarge" }
 
+[event_handler]
+# configuration for event handler which receives events from a GitHub repository.
+log_path = /path/to/eessi_bot_event_handler.log
```

Comment:
The other sections don't yet have a comment describing their purpose. So adding one here is good. However, would be better to move the comment `# configuration for ...` before `[event_handler]`.

For `log_path` there should be a short comment describing what this option is used for.